### PR TITLE
content/docs: Add FAQ content

### PR DESCRIPTION
**Pull request by cristian-vijelie**

### CODE REVIEW

**Review by razvand — APPROVED** [review]
Very good. Thanks, @cristian-vijelie 

Reviewed-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

File: content/en/docs/faq.md [hunk 1]
```diff
@@ -1,9 +1,42 @@
...
 title: FAQ
-date: 2020-01-11T14:09:21+09:00
-draft: false
-weight: 1000
 ---
 
 ## Frequently Asked Questions
 
+Q: Why Unikraft and not a container? \
+A: Unikraft is as fast and lightweight as a container, while having the security of a virtual machine.
```
Comment (nderjung) [inline content/en/docs/faq.md:8]: faster* ;)

**Review by nderjung — CHANGES_REQUESTED** [review]
In the future it would be good to have a partial that styles the list nicely, e.g.:

```
{{< faq question="Why Unikraft and not a container?" >}}
Unikraft is faster and more lightweight...
{{</ faq >}}
```

For now this is fine.  Can you just change the A to indicate being faster than a container?

**Comment by cristian-vijelie** [comment]
@nderjung done

**Comment by StefanJum** [comment]
Thank you. This will be addresed by #109, the content there is taken from here.

**Pull request closed without merging**